packages/spacecat-shared-data-access/src/models/importer/import-job.js

```diff
@@ -10,161 +10,163 @@
  * governing permissions and limitations under the License.
  */
 
+/**
+ * @typedef {import('../../').ImportJob} ImportJob
+ */
+
 import {
-  hasText, isIsoDate, isValidUrl, isObject, isString, isNumber, isInteger,
+  hasText, isIsoDate, isNumber, isInteger, isValidUrl, isString, isObject,
 } from '@adobe/spacecat-shared-utils';
 import { Base } from '../base.js';
-
-export const ImportJobStatus = {
-  RUNNING: 'RUNNING',
-  COMPLETE: 'COMPLETE',
-  FAILED: 'FAILED',
-};
+import { ImportJobStatus, ImportOptions } from './import-constants.js';
 
 /**
- * Creates a new ImportJob object.
- *
- * @param {Object} importJobData - The data for the ImportJob object.
- * @returns {ImportJob} The new ImportJob object.
+ * Creates a new ImportJob.
+ * @param data - The data for the ImportJob object.
+ * @returns {Readonly<ImportJob>} The new ImportJob object.
  */
-const ImportJob = (data) => {
-  const self = Base(data);
-
-  self.getBaseURL = () => self.state.baseURL;
-  self.getHashedApiKey = () => self.state.hashedApiKey;
-  self.getOptions = () => self.state.options;
-  self.getStartTime = () => self.state.startTime;
-  self.getEndTime = () => self.state.endTime;
-  self.getDuration = () => self.state.duration;
-  self.getStatus = () => self.state.status;
-  self.getUrlCount = () => self.state.urlCount;
-  self.getSuccessCount = () => self.state.successCount;
-  self.getFailedCount = () => self.state.failedCount;
-  self.getImportQueueId = () => self.state.importQueueId;
-  self.getInitiatedBy = () => self.state.initiatedBy;
+const ImportJobEntry = (data) => {
```

Comment:
Is this change worth it? Do all the JSDocs (like line 26 above) need to be adjusted to use this new name?